### ISSUE TITLE
Distributing TensorRT application with Windows and Linux

### ISSUE THREAD

**Issue by w3sip** [NVIDIA/TensorRT#1213]
I'm a bit confused with what is the acceptable distribution model for dependencies when it comes to TensorRT-based apps. What is the minimum set of nvidia binaries, other than TensorRT itself, that need to be included with with the software package?

For example, lets assume either Windows or Linux system, with v.455 or v.460 of driver installed. App is being distributed with TensorRT shared libs (nvinfer etc) included. What else should be?

I'm seeing that libcublas, libcublaslt and libnvrtc need to be included. So are their Windows counterparts. 
But what about libcudart? It seems required on Windows, but on not on Linux. Or libnvrtc-builtins? 

In short, is there a document someplace outlining proper way to distribute a TensorRT app?

**Comment by ttyio** [comment]
Sorry @w3sip there is no existing doc about distributing TRT app,
something might helps:
1. Windows: https://www.dependencywalker.com/
2. Linux: ldd

**Comment by ttyio** [comment]
Close since no activity for more than 3 weeks, please reopen if you still have question, thanks!
